### PR TITLE
#1 Adding time Picker at SettingWorkTime and SettingOvertime

### PR DESCRIPTION
**Pull request by fundayluck**

### CODE REVIEW

File: src/assets/svg/Settings.svg [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (LordAldi) [inline src/assets/svg/Settings.svg]: coba tag <g> kosong ini kalo dihapus aman gak

File: src/components/setting/SetOfficeLocation.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React, { useRef, useMemo, useState, useCallback } from 'react';
+import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
+
+const center = {
+  lat: 51.505,
+  lng: -0.09,
+};
```
Comment (LordAldi) [inline src/components/setting/SetOfficeLocation.jsx]: ubah ke indo aja defaultny jakarta maybe

File: src/components/setting/SetOfficeLocation.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    <div className='card'>
+      <Marker
+        draggable={draggable}
+        eventHandlers={eventHandlers}
+        position={position}
+        ref={markerRef}
+      >
+        <Popup minWidth={90}>
+          <span onClick={toggleDraggable} aria-hidden='true'>
+            {draggable ? 'Marker is draggable' : 'Click here to setting office location'}
```
Comment (LordAldi) [inline src/components/setting/SetOfficeLocation.jsx]: marker is dragable ganti ke move marker to office location

File: src/components/setting/SettingWorkTime.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+const SettingWorkTime = () => {
+  const timeValueStart = new Date('01/01/2021 08:00 AM');
+  const timeValueEnd = new Date('01/01/2021 05:00 PM');
+  const minTime = new Date('01/01/2021 08:00 AM');
+  const maxTime = new Date('01/01/2021 05:00 PM');
+  return (
+    <div className='card'>
+      <div className='card-tilte'>Work Set Time</div>
+      <div>
+        <TimePickerComponent
```
Comment (LordAldi) [inline src/components/setting/SettingWorkTime.jsx]: kasil label dong ini clockin atau clockout, disemua time picker

**Review by LordAldi — CHANGES_REQUESTED** [review]

File: src/assets/svg/Settings.svg (hunk 1 above)
Comment (fundayluck) [reply src/assets/svg/Settings.svg]: done

File: src/components/setting/SetOfficeLocation.jsx (hunk 2 above)
Comment (fundayluck) [reply src/components/setting/SetOfficeLocation.jsx]: done

File: src/components/setting/SetOfficeLocation.jsx (hunk 3 above)
Comment (fundayluck) [reply src/components/setting/SetOfficeLocation.jsx]: done

File: src/components/setting/SettingWorkTime.jsx (hunk 4 above)
Comment (jember08) [reply src/components/setting/SettingWorkTime.jsx:13]: <TimePickerComponent
          value={timeValueStart}
          min={minTime}
          max={maxTime}
          placeholder='Select a set Start Work'
          floatLabelType='clockin'
/>

Gini ga sih bang?

Comment (jember08) [reply src/components/setting/SettingWorkTime.jsx:13]: <TimePickerComponent
          value={timeValueStart}
          min={minTime}
          max={maxTime}
          placeholder='Select a set Start Work'
          floatLabelType='clockin'
        />

File: src/pages/OfficePage.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import SetOfficeLocation from '../components/setting/SetOfficeLocation';
+
+const OfficePage = () => (
+  <div className='office'>
+    <SettingWorkTime />
+    <SettingOvertime />
+    <SettingDayOff />
+    <SetOfficeLocation />
+  </div>
+);
```
Comment (LordAldi) [inline src/pages/OfficePage.jsx]:
```
const OfficePage = () => {
 const [showStartTime, setShowStartTime] = useState(false);
  const [showEndTime, setShowEndTime] = useState(false);
  const [startTime, setStartTime] = useState('');
  const [endTime, setEndTime] = useState('');
return (

  <div className='office'>
    <SettingWorkTime  showStartTime={showStartTime} setShowStartTime={setShowStartTime} ... />
    <SettingOvertime .../>
    <SettingDayOff .../>
    <SetOfficeLocation .../>
  </div>
)}
```

gini yg ak mksud, jadi statenya ada di parent trus dikirim ke component pake props

**Review by LordAldi — CHANGES_REQUESTED** [review]

File: src/pages/OfficePage.jsx (hunk 5 above)
Comment (LordAldi) [reply src/pages/OfficePage.jsx:14]: return (
  <div className='office'>
    <SettingWorkTime />
    <SettingOvertime />
    <SettingDayOff />
    <SetOfficeLocation />
  </div>
)

Comment (fundayluck) [reply src/pages/OfficePage.jsx]: okay otw

**Review by LordAldi — APPROVED** [review]
lgtm

**Pull request merged**